assets/helpers/tracking/googleTagManager.js

```diff
@@ -201,11 +201,16 @@ function gaEvent(gaEventData: GaEventData) {
   });
 }
 
+function appStoreCtaClick() {
```

Comment:
You're going to need to pass `participations` into this function and then pass it along or we won't be able to associate the clicks with a test

Reply:
This isn't a test though?

Reply:
If you do this the call sites will need to become `() => appStoreCtaClick(participations)`, because `onClick` expects a function that it can pass the event to.

Reply:
Ok, so the `appStoreCtaClick` function is passed into the `PremiumTier` and `DailyEdition` definitions having been imported at the call sites (I'm about to add it for the international variation). It's a universal call because there's no distinction between regions or app versions.

The `appStoreCtaClick` function assumes responsibility for sending `{ TestName: '' }` to `sendData` because there is no test behind this, so there's no need for a `participations` parameter in the `appStoreCtaClick` call. We don't _have_ to define a test as well, do we?

Ultimately, the only thing this has to do is trigger the `adnxs` call, which it seems to be doing locally. Is there a reason to expect it wouldn't do that in production?

Reply:
Ok so if all it's doing it triggering AppNexus that's probably ok. It just looks a bit weird having a function that tells us a button has been clicked but doesn't give any indication which one!